src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -29,40 +30,44 @@ public static bool IsGlobalExpression(this IParameter parameter)
          if (parameter == null)
             return false;
 
-         return parameter.NameIsOneOf(CoreConstants.Parameters.REL_EXP_BLOOD_CELLS,
-            CoreConstants.Parameters.REL_EXP_PLASMA, CoreConstants.Parameters.REL_EXP_VASCULAR_ENDOTHELIUM);
+         return AllGlobalRelExpParameters.Contains(parameter.Name);
       }
 
+      public static bool IsIndividualMoleculeGlobal(this IParameter parameter) =>
+         CoreConstants.Parameters.AllGlobalMoleculeParameters.Contains(parameter.Name);
+
+
       public static bool IsExpression(this IParameter parameter)
       {
          if (parameter == null)
             return false;
 
-         return parameter.IsGlobalExpression() || parameter.IsNamed(CoreConstants.Parameters.REL_EXP);
+         return parameter.IsGlobalExpression() || parameter.IsNamed(REL_EXP);
       }
 
       public static bool IsExpressionOrOntogenyFactor(this IParameter parameter)
       {
          if (parameter.IsExpression())
             return true;
 
-         if (CoreConstants.Parameters.OntogenyFactors.Contains(parameter.Name))
+         if (OntogenyFactors.Contains(parameter.Name))
             return true;
 
          return false;
       }
 
-      public static bool IsIndividualMolecule(this IParameter parameter)
+      public static bool IsExpressionProfile(this IParameter parameter)
```

Comment:
new property. LEt us know if a parameter is ExpressionProfile. If yes, we use our special command